custom_hostname_test.go

```diff
@@ -82,7 +82,63 @@ func TestCustomHostname_CreateCustomHostname(t *testing.T) {
 		assert.Equal(t, want, response)
 	}
 }
+func TestCustomHostname_CreateCustomHostname_CustomOrigin(t *testing.T) {
+	setup()
+	defer teardown()
 
+	mux.HandleFunc("/zones/foo/custom_hostnames", func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, "POST", r.Method, "Expected method 'POST', got %s", r.Method)
+
+		w.Header().Set("content-type", "application/json")
+		w.WriteHeader(http.StatusCreated)
+		fmt.Fprintf(w, `
+{
+  "success": true,
+  "errors": [],
+  "messages": [],
+  "result": {
+    "id": "0d89c70d-ad9f-4843-b99f-6cc0252067e9",
+	"hostname": "app.example.com",
```

Comment:
Can we fix up the alignment of the JSON payload here? It looks like the keys here should all be at the same level of indentation.